source/common/modules/markdown-editor/linters/language-tool.ts

```diff
@@ -85,15 +85,38 @@ const ltLinter = linter(async view => {
   const ast = markdownToAST(document)
   const textNodes = extractASTNodes(ast, 'Text') as TextNode[]
 
-  // To avoid too high loads, we have to send a "pseudo-plain text" document.
-  // That will generate a few warnings that relate towards the Markdown syntax,
-  // but we are clever: Since we can extract the textNodes, we can basically
-  // ignore any warning outside of these ranges! YAY!
+  // To avoid too high loads, we sanitize the document
+  // according to the LanguageTool API `data` parameter.
+  // By doing so, we only generate errors for TextNode regions
+  // https://languagetool.org/http-api/swagger-ui/#!/default/post_check
+  type Annotation = { text?: string, markup?: string }
```

Comment:
Suggestion: Move this to the main process and import the type here. Also: Please collect types and interfaces at the top of the files for easy access.